play-frontend-hmrc-play-30/src/main/twirl/uk/gov/hmrc/govukfrontend/views/components/GovukCheckboxes.scala.html

```diff
@@ -61,23 +61,25 @@
         <div class="govuk-checkboxes__divider">@item.divider</div>
       } else {
         @defining((values.nonEmpty && values.contains(item.value)) || item.checked) { isChecked =>
-          <div class="govuk-checkboxes__item">
-            <input class="govuk-checkboxes__input" id="@itemId(item, index)" name="@itemName(item)" type="checkbox" value="@item.value"@inputAttrs(item, index, describedBy, isChecked)@toAttributes(item.attributes)>
-              @govukLabel(Label(
-                forAttr = Some(itemId(item, index)),
-                classes = toClasses("govuk-checkboxes__label", item.label.map(_.classes).getOrElse("")),
-                attributes = item.label.map(_.attributes).getOrElse(Map.empty),
-                content = item.content
-              ))
-              @if(hasHint(item)) {
-                @govukHint(Hint(
-                  id = Some(itemHintId(item, index)),
-                  classes = toClasses("govuk-checkboxes__hint", item.hint.map(_.classes).getOrElse("")),
-                  attributes = item.hint.map(_.attributes).getOrElse(Map.empty),
-                  content = item.hint.map(_.content).getOrElse(Empty)
+          @if(item.value.nonEmpty){
```

Comment:
hmm, where did this change come from? I can't see a similar check in govuk-frontend 

https://github.com/alphagov/govuk-frontend/blame/main/packages/govuk-frontend/src/govuk/components/checkboxes/template.njk